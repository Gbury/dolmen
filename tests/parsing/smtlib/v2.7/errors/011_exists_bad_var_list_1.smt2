(set-logic ALL)
(assert (exists _ true))
