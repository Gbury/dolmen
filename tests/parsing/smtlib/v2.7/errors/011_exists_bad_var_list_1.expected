File "tests/parsing/smtlib/v2.7/errors/011_exists_bad_var_list_1.smt2", line 2, character 16-17:
2 | (assert (exists _ true))
                    ^
Error while parsing a term, read the reserved word '_',
      but expected a list of sorted variables, starting with an opening
      parenthesis.
